sortfilterproxymodel.qbs

```diff
@@ -46,6 +47,7 @@ Product
             return defines
         }
 
+        cpp.linkerFlags: "--no-as-needed"
```

Comment:
Not sure, I'll have to look at this more closely. Can you provide any more information regarding the runtime error that occurred? Maybe `ldd` and/or `strace` output, from both before and after this change?